src/test/java/uk/gov/pay/adminusers/resources/InviteResourceOtpIT.java

```diff
@@ -270,4 +270,46 @@ public void validateServiceOtpKey_shouldFailWith401_whenInvalidOtp() throws Exce
                 .then()
                 .statusCode(UNAUTHORIZED.getStatusCode());
     }
+
+    @Test
+    public void validateOtpKey_shouldSucceed_whenValidOtp() throws Exception {
+
+        code = InviteDbFixture.inviteDbFixture(databaseHelper)
+                .withOtpKey(OTP_KEY)
+                .insertInvite();
+
+        Map<Object, Object> sendRequest = Map.of(
+                "code", code,
+                "otp", PASSCODE);
+
+        givenSetup()
+                .when()
+                .body(mapper.writeValueAsString(sendRequest))
+                .contentType(JSON)
+                .post(V2_INVITES_VALIDATE_OTP_RESOURCE_URL)
+                .then()
+                .statusCode(OK.getStatusCode());
+    }
+
+    @Test
+    public void validateOtpKey_shouldFailWith401_whenInvalidOtp() throws Exception {
```

Comment:
It looks like there wasn't an equivalent test for `/v1/otp/validate/service` like validateOtp_shouldFailAndLockInvite_whenInvalidOtpAuthCode_ifMaxRetryExceeded`.

Is this adequately covered by unit tests, or is it worth adding an equivalent test for the v2 endpoint?

Reply:
I think it makes sense to add an integration test.